frontend/src/metabase/collections/containers/CollectionSidebar/CollectionSidebar.jsx

```diff
@@ -79,90 +72,46 @@ class CollectionSidebar extends React.Component {
-                    to={Urls.collection({ id: "root" })}
-                    selected={isRoot}
-                    highlighted={highlighted}
-                    hovered={hovered}
-                  >
-                    {t`Our analytics`}
-                  </CollectionLink>
-                )}
-              </CollectionDropTarget>
-            </Box>
-          )}
+          {({ collection: root }) => <Header isRoot={isRoot} root={root} />}
         </Collection.Loader>
 
-        <Box pb={4}>
-          <CollectionsList
-            openCollections={this.state.openCollections}
-            onClose={this.onClose}
-            onOpen={this.onOpen}
-            collections={list}
-            filter={nonPersonalOrArchivedCollection}
-            currentCollection={collectionId}
-          />
-
-          <Box>
-            <CollectionsList
-              openCollections={this.state.openCollections}
-              onClose={this.onClose}
-              onOpen={this.onOpen}
-              collections={currentUserPersonalCollections(list, currentUser.id)}
-              initialIcon="person"
-              filter={this.filterPersonalCollections}
-              currentCollection={collectionId}
-            />
-          </Box>
-        </Box>
-
-        <Box className="mt-auto" pb={2} pl={SIDEBAR_SPACER * 2}>
-          {currentUser.is_superuser && (
-            <Link
-              my={2}
-              to={Urls.collection({ id: "users" })}
-              className="flex align-center text-bold text-light text-brand-hover"
-            >
-              <Icon name="group" mr={1} />
-              {t`Other users' personal collections`}
-            </Link>
-          )}
-          <Link
-            to={`/archive`}
-            className="flex align-center text-bold text-light text-brand-hover"
-          >
-            <Icon name="view_archive" mr={1} />
-            {t`View archive`}
-          </Link>
-        </Box>
+        <Collections
+          collectionId={collectionId}
+          currentUserId={currentUser.id}
+          list={list}
+          onClose={this.onClose}
+          onOpen={this.onOpen}
+          openCollections={this.state.openCollections}
+        />
+
+        <Footer isSuperUser={currentUser.is_superuser} />
       </React.Fragment>
     );
   };
 
   render() {
-    const { allFetched } = this.props;
+    const { allFetched, shouldDisplayMobileSidebar } = this.props;
 
     return (
       <Sidebar
         role="tree"
-        shouldDisplayMobileSidebar={this.props.shouldDisplayMobileSidebar}
+        shouldDisplayMobileSidebar={shouldDisplayMobileSidebar}
       >
-        {allFetched ? (
-          this.renderContent()
-        ) : (
-          <div className="text-brand text-centered">
-            <LoadingSpinner />
-            <h2 className="text-normal text-light mt1">{t`Loading…`}</h2>
-          </div>
-        )}
+        {allFetched ? this.renderContent() : <LoadingWarning />}
```

Comment:
"Warning" makes me think something is off, but here we just want to show a spinner
wdyt about using smth like `LoadingView` / `LoadingState` / `Spinner`?